akka-stream/src/main/scala/akka/stream/impl/ActorRefBackpressureSinkStage.scala

```diff
@@ -32,7 +32,7 @@ import akka.stream.stage._
     new GraphStageLogic(shape) with InHandler {
       implicit def self: ActorRef = stageActor.ref
 
-      val maxBuffer = inheritedAttributes.getAttribute(classOf[InputBuffer], InputBuffer(16, 16)).max
+      val maxBuffer = inheritedAttributes.get(classOf[InputBuffer], InputBuffer(16, 16)).max
```

Comment:
wouldn't `inheritedAttributes.get[InputBuffer](InputBuffer(16, 16)).max` solve the problem without adding more methods to the API?

See
```
def get[T <: Attribute: ClassTag](default: T): T
```

Reply:
strictly correct, fixed and squashed commits. :+1: